README.md

```diff
@@ -24,14 +24,15 @@ pip install mirdata
 
 Try it out!
 ```python
-import mirdata.orchset
+import mirdata
 import random
 
-mirdata.orchset.download()  # download the dataset
-mirdata.orchset.validate()  # validate that all the expected files are there
-orchset_data = mirdata.orchset.load()  # (lazy) load the data index
+orchset = mirdata.Dataset('orchset')
+orchset.download()  # download the dataset
+orchset.validate()  # validate that all the expected files are there
+orchset_data = orchset.load_tracks()  # (lazy) load the data index
```

Comment:
remove (lazy)? will see if we remove this completely later

Reply:
also we're changing load --> load_tracks? what happens in the case of recommendation datasets or multitrack? better to keep load?

Reply:
I was thinking in order to keep everything consistent, we could have `load_tracks`, `load_multitracks` and `load_tables`, which will always be available as methods but might raise an error when dataset doenst have that kind of data.

You can imagine a dataset which has both a table and tracks, or both tracks and multitracks, etc. By breaking it up, everything stays clean

Reply:
Like it!

Reply:
me too!